src/snowflake/snowpark/files.py

```diff
@@ -93,58 +93,119 @@ def close(self) -> None:
         return
 
     def detach(self) -> None:
+        """
+        Not yet supported in UDF and Stored Procedures.
+
+        See https://docs.python.org/3/library/io.html#io.BufferedIOBase.detach
+        """
         raise NotImplementedError(_DEFER_IMPLEMENTATION_ERR_MSG)
 
     def fileno(self) -> None:
+        """
+        Not yet supported in UDF and Stored Procedures.
+
+        See https://docs.python.org/3/library/io.html#io.IOBase.fileno
+        """
```

Comment:
Based on SfStream implementation, this actually returns `OSError("This object does not use a file descriptor")`

Reply:
What would you recommend? I was thinking that not supported is simple enough and they probably don't care what it actually fails with? Should we change it to unsupported? Or update it to "Returns os error"?